app/package.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
it is for oc to build and deploy. s2i uses `npm install`, `npm build, and `npm start` on the `contextDir` folder